tools/nni_cmd/nnictl_utils.py

```diff
@@ -700,11 +700,11 @@ def export_trials_data(args):
                 trial_records = []
                 for record in content:
                     if not isinstance(record['value'], (float, int)):
-                        formated_record = {**record['parameter'], **record['value'], **{'id': record['id']}}
+                        formated_record = {**record['parameter'], **json.loads(record['value']), **{'id': record['id']}}
```

Comment:
maybe we should use `json_tricks` instead of `json` in this file?

Reply:
for example `NaN`

Reply:
@QuanluZhang there are still many other usage of `json` package in `nnictl_utils.py`. Do you want to replace all of them?

Reply:
I think it is okay to keep them as long as `json.load` is not used on `value`. Could you help give a quick check?

Reply:
seem no drawback to replace them, decided on you :)